tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture
+def docker_env_vars(monkeypatch):
+    """Monkeypatch environment variables that we'd get running under docker."""
+    monkeypatch.setenv('PHABRICATOR_URL', 'http://phabricator.test')
```

Comment:
If you're going to change the phab url, can you also go through and update all the canned responses to use this URL? Right now they use the default mozphab.aws URL which came from the original docker-compose.yml.